README.md

```diff
@@ -13,17 +15,34 @@ For more information about contributing to the Kubernetes documentation, see:
 * [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
 * [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
 
+## Running the site locally using Hugo
+
+The Kubernetes documentation is built using the [Hugo](https://gohugo.io) static site generator. See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions.
+
+> The site currently requires Hugo version **0.46**.
+
+To run the site locally when you have Hugo installed:
+
+```bash
+make serve
+
+# The underlying command
+hugo server --ignoreCache --disableFastRender
+```
+
+This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the site. As you make changes to docs' source files, Hugo will update immediately and force a browser refresh.
+
 ## Building the site using Docker
 
 If you'd like, you can build the Kubernetes docs using Docker. To get started, build the image locally:
 
 ```bash
 make docker-image
```

Comment:
Does this mean that the `make` command is actually doing the underlying command? The way this is presented might lead people to think they need to run both commands.

Reply:
Good call. I've removed the underlying command and kept only the `make` command. Users can investigate the underlying commands on their own.